ci/scripts/run-codebuild/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "run-codebuild",
+  "version": "0.0.1",
+  "description": "Run codebuild script",
+  "type": "module",
+  "scripts": {
+    "test": "jest --silent=true",
+    "lint": "standard"
+  },
```

Comment:
Optional, but we could add an `engines` attribute here to manage expectations of what the code can run against, something like

```
 "engines": {
    "node": ">=12.22.7 <15"
  },
```

Reply:
I could just do >=12.22.7, I've tested it all the way to node17 and everything works without problem so we don't currently have an upper bound

Reply:
Done :)